src/index.html

```diff
@@ -3,11 +3,348 @@
+              <div class="grid">
+                <div class="grid__item grid__item--desktop-1-4 grid__item--tablet-1-2">
+                  <div class="place__photo-container">
+                    <img
+                      src="images/photos/hydraulic.png"
+                      alt="Hydraulic disc brakes and Lightweight 1"
+                      class="place__photo"
+                    >
+                  </div>
+                </div>
+                <div class="grid__item grid__item--desktop-5-12 grid__item--tablet-3-6">
+                  <div class="place__photo-container">
+                    <img
+                      src="images/photos/lightweight.png"
+                      alt="Hydraulic disc brakes and Lightweight 2"
+                      class="place__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="place__description">
+              <h3 class="place__title">Hydraulic disc brakes & Lightweight</h3>
+              <div class="grid grid--block">
+                <div class="grid__item grid__item--desktop-1-8 grid__item--tablet-1-6">
+                  <p class="place__info">
+                    Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="grid grid--block">
+            <div class="grid__item grid__item--tablet-2-5 grid__item--desktop-5-9">
+              <a href="/" class="button">
+                Explore
+              </a>
+            </div>
+          </div>
+
+        </div>
+      </section>
+      <section class="main__section main__section--bikes bikes" id="contacts">
+        <div class="container">
+          <h2 class="main__section-title">
+            Contact us
+          </h2>
+          <div class="grid grid--block">
+            <div class="grid__item grid__item--tablet-1-3 grid__item--desktop-1-5">
+              <form action="#">
+                <input
+                  type="text"
+                  name="name"
+                  class="form-field contact__field"
+                  placeholder="Name"
+                  required
+                >
+                <input
+                  type="email"
+                  name="email"
+                  class="form-field contact__field"
+                  placeholder="Email"
+                  required
+                >
+                <textarea
+                  name="message"
+                  class="form-field contact__field contact__field--last form-field--textarea"
+                  placeholder="Message"
+                  required
+                ></textarea>
+                <button class="button contact__button">
+                  Send
```

Comment:
This form should have a submit button (type='submit')
+ checklist #15:
Page shouldn't be reloaded on form submit (https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault)

Reply:
thx, i fixed it